cmd/snap/cmd_prepare_image.go

```diff
@@ -145,6 +148,16 @@ func (x *cmdPrepareImage) Execute(args []string) error {
 		opts.Customizations = *custo
 	}
 
+	if x.Validation != "" {
+		if opts.Customizations.Validation == "" {
+			opts.Customizations.Validation = x.Validation
+		} else {
+			opts.Customizations = image.Customizations{
+				Validation: x.Validation,
+			}
+		}
```

Comment:
hm this doesn't look right, there could be other options set in customiazations, would this work? giving the priority to `--validation` 

```suggestion
		// validation passed in the command line overrides customizations
		opts.Customizations.Validation = x.Validation
```